src/client/components/AddImageToRoom/AddImageToRoom.styles.css

```diff
@@ -0,0 +1,35 @@
+.add-image-to-room input {
+  display: none;
+}
+
+.image-container {
```

Comment:
I suggest all selectors start with `.add-image-to-room ` to properly scope this.